docs/user/cli.md

```diff
@@ -142,73 +149,46 @@ When the preconditions for the installation are not met, it informs the user and
 
 
 
-## `agama profile`
-
-Manage auto-installation profiles (retrieving, applying, etc.)
-
-**Usage:** `agama profile <COMMAND>`
-
-###### **Subcommands:**
-
-* `autoyast` — Download the autoyast profile and print resulting json
-* `validate` — Validate a profile using JSON Schema
-* `evaluate` — Evaluate a profile, injecting the hardware information from D-Bus
-* `import` — Process autoinstallation profile and loads it into agama
-
-
-
-## `agama profile autoyast`
-
-Download the autoyast profile and print resulting json
-
-**Usage:** `agama profile autoyast <URL>`
-
-###### **Arguments:**
-
-* `<URL>` — AutoYaST profile's URL. Any AutoYaST scheme, ERB and rules/classes are supported. all schemas that autoyast supports
-
-
-
-## `agama profile validate`
+## `agama config validate`
 
 Validate a profile using JSON Schema
 
 Schema is available at /usr/share/agama-cli/profile.schema.json
 
-**Usage:** `agama profile validate <URL_OR_PATH>`
+**Usage:** `agama config validate <URL_OR_PATH>`
 
 ###### **Arguments:**
 
 * `<URL_OR_PATH>` — JSON file, URL or path or `-` for standard input
 
 
+## `agama config generate`
```

Comment:
I don't fully understand how this command works.

Does it take any AutoYaST/Agama profile as input and generates the Agama config as output?

Does it export the current config into all those formats (I don't think so, but..)?

Reply:
It turns an AutoYaST or an Agama Jsonnet/JSON file and generates de configuration output. A few notes:

* If given a URL, it downloads the file. Bear in mind that in AutoYaST, downloading the profile could imply more than one file (e.g., rules/classes).
* For a JSON file, it just downloads the file.
* Reading from the stdin prevents downloading any file. It might be useful to convert an single-file AutoYaST profile or a Jsonnet one.

Reply:
I understood it after writing my comment. I guess that "AutoAgama" will pipe a "config generate" to a "config load".

Reply:
So that current internal conversion becomes somehow more explicit and/or accessible.

Reply:
Yes, that's the point, to make the conversion more explicit. However, we cannot do `agama config generate URL | agama config load` because we might need to validate the result in the middle. But that's a different topic.

Reply:
>  However, we cannot do agama config generate URL | agama config load because we might need to validate the result in the middle.  But that's a different topic.

It is quite relevant. I am leaning toward making validation automatic:
- automatic and mandatory on input (`load`, `edit`)
- automatic but optional (`--no-validate`) for output (`generate`, `show`). Remember how AY would export profiles that would not validate? Not Agama.
- explicit by `validate` (which in light of the above is just `config blackhole` but with an automatic validation part)

This way, `agama config generate URL | agama config load` performs validation twice, but I think it is a small price to pay for the resulting robustness for users and developers.